cmd/root.go

```diff
@@ -65,6 +65,13 @@ func init() {
 	viper.BindPFlag("kong-addr",
 		rootCmd.PersistentFlags().Lookup("kong-addr"))
 
+	rootCmd.PersistentFlags().StringSlice("headers", []string{},
+		"HTTP Headers to call Kong's Admin API.\n"+
```

Comment:
Let's change this to `HTTP Headers(key:value) to inject in all requests to Kong's Admin API.\n. This flag can be specified multiple times to inject multiple headers.`

Reply:
Definitely a better message 👍